nym-wallet/src/context/bonding.tsx

```diff
@@ -165,6 +181,26 @@ export const BondingContextProvider = ({ children }: { children?: React.ReactNod
     return undefined;
   };
 
+  const getSetProbabilities = async (mixId: number) => {
+    let result;
+    try {
+      result = await getInclusionProbability(mixId);
+    } catch (e: any) {
+      Console.log(e);
+    }
+    return result;
+  };
```

Comment:
```
  const getSetProbabilities = async (mixId: number) => {
    try {
      const result = await getInclusionProbability(mixId);
      return result
    } catch (e: any) {
      Console.log(e);
    }
  };
```
Can do it this way?

Reply:
Yea, but doing this make ESLint not happy "Expected to return a value at the end of async arrow function. (eslint)"